swift/Diffuser/Sources/Diffuser/diffuser/Combinators.swift

```diff
@@ -31,6 +31,38 @@ public extension Diffuser {
         return Diffuser(effect: effect)
     }
 
+    /// Create a Diffuser which merges a list of Diffusers and always executes them regardless of the value.
+    ///
+    /// This function is useful as a building block for more complex Diffusers, but it is unlikely that you would use
+    /// it on its own. Consider using `into` instead.
```

Comment:
Is this sentence still relevant in those two new variants of `intoAlways`?

Reply:
Also, perhaps you could document briefly the intended use case of `intoAlways` for non-Equatable models with Equatable members.

Reply:
Updated! I think that sentence would still apply in this case but it's not really that clear so I replaced it to point out about the existence of `intoAll` instead.